test/e2e/app-dir/next-after-app/index.test.ts

```diff
@@ -58,6 +59,10 @@ describe.each(runtimes)('unstable_after() in %s runtime', (runtimeValue) => {
   })
 
   const getLogs = () => {
+    if (next.cliOutput.length < currentCliOutputIndex) {
+      // cliOutput shrank since we started the test, so something (like a `sandbox`) reset the logs
+      currentCliOutputIndex = 0
+    }
```

Comment:
this is a bit of a hack, and it'd be cleaner to replace this with something like a
```
beforeEach(() => {
  next.clearCliOutput() // method doesn't exist yet
})
```
but i'll do that in a follow up